content/ui-patterns/notification-messaging.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Notification messaging
+description: Messaging components are used to provide important and relevant information to the user, including feedback, contextual information, product updates, and more.
+---
+
+import { Box } from '@primer/react'
+import { InfoIcon, CheckCircleIcon, StopIcon, AlertIcon } from '@primer/octicons-react'
+
+Primer messaging components:
+
+- Announcement (not yet available)
```

Comment:
Wanted to ask where is the best place to look to learn more about Announcement 👀